code/datums/gamemodes/gangwar.dm

```diff
@@ -242,20 +268,70 @@
 
 	return
 
-/datum/game_mode/gang/proc/process_hot_zones()
-	hot_zone = pick(potential_hot_zones)
+/datum/game_mode/gang/proc/process_lootcrate_spawn()
+	drop_zone = pick(potential_hot_zones)
+	var/turfList[0]
+	var/valid = 0
+
+	for (var/turf/simulated/floor/T in drop_zone.contents)
+		if (!T.density)
+			valid = 1
+			for (var/obj/O in T.contents)
+				if (O.density)
+					valid=0
+					break
+				else
+			if (valid == 1)
+				turfList.Add(T)
 
-	broadcast_to_all_gangs("The [hot_zone.name] is a high priority area. Ensure that your gang has control of it five minutes from now!")
+	var/turf/target = pick(turfList)
+	new/obj/storage/crate/gang_crate/guns_and_gear(target)
+	broadcast_to_all_gangs("We've dropped off weapons & ammunition at the [drop_zone.name]! It's anchored in place for 5 minutes, so get fortifying!")
 
-	SPAWN(hot_zone_timer - 1 MINUTE)
-		if(hot_zone != null) broadcast_to_all_gangs("You have a minute left to control the [hot_zone.name]!")
+
+	SPAWN(crate_drop_timer-600)
+		if(drop_zone != null)
+			broadcast_to_all_gangs("The weapons crate at the [drop_zone.name] can be moved in 1 minute!")
 		sleep(1 MINUTE)
-		if(hot_zone != null && hot_zone.gang_owners != null)
-			var/datum/gang/G = hot_zone.gang_owners
-			G.score_event += hot_zone_score
-			broadcast_to_all_gangs("[G.gang_name] has been rewarded for their control of the [hot_zone.name].")
-			sleep(10 SECONDS)
-		process_hot_zones()
+		if(drop_zone != null)
+			broadcast_to_all_gangs("The weapons crate at the [drop_zone.name] is free! Drag it to your locker!")
+		sleep(crate_drop_timer)
+		process_lootcrate_spawn()
```

Comment:
Pretty iffy doing infinite recursion?

Reply:
yea its the same as the old code, should be updated